docs/contributing/contributing-code/contributing-code-tests/testing-local.md

```diff
@@ -84,3 +85,46 @@ go run ./cmd/rad/main.go workspace delete
 Another approach would be to delete and recreate the kubernetes cluster
 
 Pro tip: `kubectl config set-context --current --namespace=radius-system` to the radius-system namespace
+
+## Recipe Testing: 
+(@vishwa, @karishma)
+### Bicep Recipes: 
+### Terraform Recipes:
+- One way to create and test recipes locally is to create a a public/private git repo to host recipes and access/test it setting it's template path to the git repo location. eg:
+
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+ ...
+        recipes: {
+      'Applications.Core/extenders': {
+        default: {
+          templateKind: 'terraform'
+          templatePath:'git::https://github.com/<username>/<reponame>'
+        }
+      }
+    }
+ ...   
+
+
+- Another way to test recipes is to convert a folder containing recipes into a http server that serves that folder locally and hosts recipes.
+  ``` 
+  python command: python3 -m http.server
+  ```
+
+## CoreRP Testing:
+(@yetkin) - How to use cosmos db as a backend where one can see individual entries stored in db.
+          - Use/share Postman Collection to test api
+
+
+## Preferred Tools used to visualize k8s logs (pros/cons)
+(@yetkin/@vishwa) 
```

Comment:
I'm a big fan of stern. https://github.com/stern/stern

- `stern '.*' -n radius-system` outputs all logs from Radius components
- `stern 'ucpd.*' -n radius-system` outputs UCP logs.

Reply:
It would also be nice to have some non-JSON options for log viewing. JSON logs are good for machines and bad for humans